build/zig/build-0.14.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2024 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=zig
+VER=0.14.0
+MAJVER=${VER%%.*}
+MINVER=${VER%.*}
+MINVER=${MINVER#*.}
+PKG=ooce/developer/zig-014
+SUMMARY="$PROG programming language"
+DESC="$PROG is a general-purpose programming language and toolchain for "
+DESC+="maintaining robust, optimal, and reusable software."
+
+min_rel 151049
```

Comment:
we only ship llvm/clang 19 from r151051 on. this build will likely break on r150150 which is still supported. the barrier should probably be r52